aten/src/ATen/native/cudnn/Macros.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <ATen/cudnn/cudnn-wrapper.h>
+
+// Warning: The following macro is guard for enabling cuDNN v8 API,
+// which is not finished yet on PyTorch. This macro will be removed
+// once the cuDNN v8 binding is done.
+// cuDNN v8 API is not finished yet, and is not recommended to use.
+// enable this only if you know what you are doing.
+#define _ENABLE_CUDNN_V8_API true
```

Comment:
I will change this to `false` before merge
```suggestion
#define _ENABLE_CUDNN_V8_API false
```